aidants_connect_web/models.py

```diff
@@ -561,6 +561,31 @@ def duration_for_humans(self) -> int:
         # and 0 for a computer.
         return duration_for_computer.days + 1
 
+    @property
+    def should_display_revocation_attestation(self) -> bool:
+        return (
+            self.is_revoked
+            and not self.mandat.was_explicitly_revoked
+            and self.was_explicitely_revoked
+        )
+
+    @cached_property
+    def was_explicitely_revoked(self) -> bool:
```

Comment:
Que dirais-tu d'une reformulation pour autodocumenter tout ça un peu mieux ?

```suggestion
    def was_separately_revoked(self) -> bool:
```